### PR TITLE
Add autopep utility

### PR DESCRIPTION
**Pull request by JasonMWhite**
This PR introduces a replacement for the "autopep" portion of the github pre-push hook. It's pretty simple, just calls the `autopep8.fix_multiple_files` with appropriate options to fix files in-place, using the same settings as starscream and data_bang currently use.

I've made the interface a little more generic so we can choose to expose additional options in the future if necessary.

@cfournie What do you think?

### CODE REVIEW

File: shopify_python/git_utils.py [hunk 1]
```diff
@@ -48,3 +50,47 @@ def changed_python_files_in_tree(root_path):
...
+                              global_config=None,
+                              ignore='',
+                              ignore_local_config=False,
+                              in_place=True,
+                              indent_size=4,
+                              jobs=0,
+                              line_range=None,
+                              list_fixes=False,
+                              max_line_length=max_line_length,
+                              pep8_passes=-1,
```
Comment (cfournie) [inline shopify_python/git_utils.py]: What does this setting mean?

File: shopify_python/git_utils.py [hunk 2]
```diff
@@ -48,3 +50,47 @@ def changed_python_files_in_tree(root_path):
...
+                              recursive=False,
+                              select={'W', 'E'},
```
Comment (cfournie) [inline shopify_python/git_utils.py]: What does this setting mean?

Comment (JasonMWhite) [inline shopify_python/git_utils.py]: Most of them are documented on the `autopep` command line docs. I'll put in a link.

Comment (JasonMWhite) [inline shopify_python/git_utils.py]: This particular setting means "fix all issues whose code starts with a W or E". Which is all of them.

**Pull request merged**